components/brave_shields/ios/browser/brave_shields_utils_ios+private.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_BRAVE_SHIELDS_IOS_BROWSER_BRAVE_SHIELDS_UTILS_IOS_PRIVATE_H_
+#define BRAVE_COMPONENTS_BRAVE_SHIELDS_IOS_BROWSER_BRAVE_SHIELDS_UTILS_IOS_PRIVATE_H_
+
+#import <Foundation/Foundation.h>
+
+#include "brave/components/brave_shields/ios/browser/brave_shields_utils_ios.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+class HostContentSettingsMap;
+class PrefService;
+
+@interface BraveShieldsUtilsIOS (Private)
+- (instancetype)initWithHostContentSettingsMap:(HostContentSettingsMap*)map
+                                    localState:(PrefService*)localState
+                                  profilePrefs:(PrefService*)profilePrefs;
+@end
```

Comment:
This no longer needs to be in a seprate header or private, since the `brave_shields_utils_ios.h` file itself is not public